tests/unit_tests/storage/test_local_storage.py

```diff
@@ -643,9 +643,9 @@ def get_experiment(self, model_experiment: Experiment):
             list(storage_experiment.response_configuration.values())
             == model_experiment.responses
         )
-        assert model_experiment.observations == pytest.approx(
```

Comment:
But it also previously did a full comparison of datasets? How can it then go from ~ 1 minute to ~ 1 hour? Or is it compounding errors?

Reply:
Not sure exactly how or why, though reducing data has also made it faster so most likely this is/should not be a problem, ref this profiling shows that the comparison was like ~40% of the time spent, though it is scaled by the amount of data generated. 
<img width="1244" alt="Screenshot 2024-05-28 at 13 32 03" src="https://github.com/equinor/ert/assets/11595637/2a20b9ad-83ad-479c-ac26-8aae7bb7939d">